internal/metrics/handler.go

```diff
@@ -0,0 +1,77 @@
+package metrics
+
+import (
+	"strings"
+
+	"github.com/VictoriaMetrics/metrics"
+	"github.com/labstack/echo"
+)
+
+type MetricsConfig struct {
+	Prometheus PrometheusConfig
+}
+
+type PrometheusConfig struct {
+	Enabled bool   `koanf:"enabled"`
+	Path    string `koanf:"path"`
+}
+
+func HandlePromMetrics() echo.HandlerFunc {
+	return echo.HandlerFunc(
+		func(c echo.Context) error {
+			metrics.WritePrometheus(c.Response().Writer, true)
```

Comment:
Whether to expose Go process metrics or not, can also come as config instead of passing a hardcoded `true`.

Reply:
Oh, you mean we always register the handler, but it it's disabled it's just an empty handler?

That would be nicer to read.